The failure is unrelated to my patch, it is because the nightly repo for Trace compass is no longer available.

```
[ERROR] Internal error: java.lang.RuntimeException: Failed to resolve target definition /home/travis/build/sshahriyar/totalads/org.eclipse.tracecompass.totalads.target/totalads-4.4.target: Failed to load p2 metadata repository from location http://download.eclipse.org/tracecompass/master/nightly: No repository found at http://download.eclipse.org/tracecompass/master/nightly. -> [Help 1]
```

I'll submit another commit that fixes that.

--- a/org.eclipse.tracecompass.totalads.target/totalads-4.4.target
+++ b/org.eclipse.tracecompass.totalads.target/totalads-4.4.target
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<?pde version="3.8"?><target name="totalads-4.4" sequenceNumber="56">
+<?pde version="3.8"?><target name="totalads-4.4" sequenceNumber="58">
 <locations>
 <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
 <unit id="org.eclipse.license.feature.group" version="0.0.0"/>
@@ -17,7 +17,7 @@
 <unit id="org.eclipse.tracecompass.tmf.feature.group" version="0.0.0"/>
 <unit id="org.eclipse.tracecompass.lttng2.kernel.feature.group" version="0.0.0"/>
 <unit id="org.eclipse.tracecompass.testing.feature.group" version="0.0.0"/>
-<repository location="http://download.eclipse.org/tracecompass/master/nightly"/>
+<repository location="http://download.eclipse.org/tracecompass/master/repository"/>
 </location>
 <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
 <unit id="org.eclipse.cdt.gnu.dsf.feature.group" version="0.0.0"/>
@@ -63,7 +63,7 @@
 <unit id="org.eclipse.tracecompass.btf.feature.group" version="0.0.0"/>
 <unit id="org.eclipse.tracecompass.tmf.pcap.feature.group" version="0.0.0"/>
 <unit id="org.eclipse.tracecompass.lttng2.kernel.feature.group" version="0.0.0"/>
-<repository location="http://download.eclipse.org/tracecompass/master/nightly"/>
+<repository location="http://download.eclipse.org/tracecompass/master/repository"/>
 </location>
 <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
 <unit id="org.eclipse.remote.feature.group" version="1.1.0.201409111520"/>

